lya_cosmo/py/camb_cosmo.py

```diff
@@ -136,8 +136,10 @@ def get_camb_results(pars,zs=None,fast_camb=False):
         else:
             kmax_Mpc=camb_kmax_Mpc
 
+        h=pars.H0/100.0
+
         # camb_extra_kmax will allow to evaluate the power later on at kmax
-        pars.set_matter_power(redshifts=zs,kmax=camb_extra_kmax*kmax_Mpc,
+        pars.set_matter_power(redshifts=zs,kmax=(camb_extra_kmax*kmax_Mpc)/h,
```

Comment:
I'm not sure about this, I'm pretty sure that the argument should be in inverse Mpc.

Reply:
Ah dammit, I was using an older version of `CAMB=1.0.3` on my laptop when working on this. On my desktop on `CAMB=1.1.3` I don't get this problem when working on master. I think the units were perhaps different in older version. I'll revert this and update the doc.

Reply:
Which `CAMB` version are you working on? Just to verify this

Reply:
Ok I've reverted this commit